{{cookiecutter.project_name}}/.github/workflows/release.yaml

```diff
@@ -19,11 +19,8 @@ jobs:
         with:
           filter: blob:none
           fetch-depth: 0
-      - uses: actions/setup-python@v5
-        with:
-          python-version: "3.x"
-          cache: "pip"
-      - run: pip install build
-      - run: python -m build
+      - name: Install Hatch
+        uses: pypa/hatch@install
+      - run: hatch build
```

Comment:
Not sure if we have a good reason to switch to hatch here, but it would be possible.

Reply:
As said above, I probably wouldn’t, but using `pyproject-build --installer=uv` would make this very slightly faster

Reply:
Since we already go all in with hatch, it's fine with me.